compiler/src/test/java/io/neow3j/compiler/PlaceholderSubstitutionTest.java

```diff
@@ -33,16 +33,16 @@ public static void main() {
     @Test
     public void testMultiAnnotationReplacement() throws IOException {
         Map<String, String> replaceMap = new HashMap<>();
-        replaceMap.put("INVALID_TRUST_HASH1", "*");
-        replaceMap.put("INVALID_TRUST_HASH2", "*");
+        replaceMap.put("INVALID_TRUST_HASH1", "0xef4073a0f2b305a38ec4050e4d3d28bc40ea63f5");
+        replaceMap.put("INVALID_TRUST_HASH2", "0xd2a4cff31913016155e38e474a2c06d08be276cf");
```

Comment:
Afaik, these are valid hashes, right? Then, can we rename the string to something else, e.g., `TRUST_HASH1`?

Reply:
thanks, i've missed that.